recipes/TXM_Sandbox/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TXM_Sandbox" %}
+{% set version = "0.1.0" %}
```

Comment:
Use the same version as pypi: `0.1`

Reply:
A new version 0.1.1 is updated in pypi in which LICENSE file is included with the suggested MANIFEST.in approach. The version in the conda recipe is changed to 0.1.1 accordingly.